### PR TITLE
add sticky help message to get started

### PR DESCRIPTION
**Pull request by bishabosha**
Add a toggle details to point users to Discord if they have problems with set up. The details will stick at the top of the screen as you scroll. It also has a more red theme to match its importance level. Also toggle works without JS.

first screenshot shows page at load:

Screenshot 2022-06-08 at 15 55 40

next screenshot demonstrates what it looks like when expanded and sticky while scrolling:

Screenshot 2022-06-08 at 15 53 39

### CODE REVIEW

**Review by sjrd — CHANGES_REQUESTED** [review]

File: _includes/getting-started.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 The instructions below cover both Scala 2 and Scala 3.
 
+<div class="inline-sticky-top">
+{% altDetails need-help-info-box 'Need Help?' class=help-info %}
+*If you are having trouble with setting up Scala, feel free to ask for help in the [#scala-users](https://discord.com/channels/632150470000902164/632150470000902166) channel of our Discord.*
```
[the remark this answers is not in the document]
Comment (bishabosha) [reply _includes/getting-started.md:5]: should we have two links (one for invite, one for the channel) or can you invite and forward to a channel in one link?

[the remark this answers is not in the document]
Comment (sjrd) [reply _includes/getting-started.md:5]: I don't think there is a way to invite+send to specific channel. Usually we use the invite link, and mention in words to go to the specific channel.

**Review by julienrf — APPROVED** [review]
Thank you Jamie, that looks great!

File: _includes/getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bishabosha) [reply _includes/getting-started.md]: changed the link

**Pull request merged**